server/sse.go

```diff
@@ -21,9 +21,9 @@ import (
 	"github.com/openfresh/plasma/pubsub"
 )
 
-func NewSSEServer(pb pubsub.PubSuber, accessLogger *zap.Logger, errorLogger *zap.Logger, config config.Config) *http.Server {
+func NewSSEServer(opt Option) *http.Server {
```

Comment:
__[golint]__
* [golint] exported function NewSSEServer should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->